compile.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
+
+set -e
+
```

Comment:
Oh yeah – http://redsymbol.net/articles/unofficial-bash-strict-mode/
```bash
#!/bin/bash
set -euo pipefail
IFS=$'\n\t'
```
I start all scripts that way.